### PR TITLE
fix Issue 24187 - ImportC: _Float32 not defined

### PR DESCRIPTION
**Pull request by WalterBright**
I added `_Float64` just because.

### CODE REVIEW

**Comment by thewilsonator** [comment]
please include the bug reports example as a test case. Also this doesn't appear to handle `_Float32x` (I don't know what that is). 

Also also, why is this linux/(not AArch64) exclusive?

Also bug is labeled blocker, which while it is not that serious, it could go to stable if it is `importc.h` changes only with no risk of regression.

**Comment by WalterBright** [comment]
Huh, I didn't notice the `x`.

**Comment by WalterBright** [comment]
> why is this linux/(not AArch64) exclusive?

Because a previous report put these _Float things under linux, and the bug report was marked linux.

File: druntime/src/importc.h [hunk 1]
```diff
@@ -152,6 +152,10 @@
 #define __PRETTY_FUNCTION__ __func__
 
 #ifndef __aarch64__
+#define _Float32 float
+#define _Float32x float
```
Comment (ibuclaw) [inline druntime/src/importc.h:156]: Does nobody read the spec? 

C23 - H.2.3 Extended floating types
> For each of its basic formats, IEC 60559 specifies an extended format whose maximum exponent and
precision exceed those of the basic format it is associated with. Extended formats are intended for
arithmetic with more precision and exponent range than is available in the basic formats used for
the input data. The extra precision and range often mitigate round-off error and eliminate overflow
and underflow in intermediate computations. The table below gives the minimum values of these
parameters, as defined for the C floating-point model (5.2.4.2.2). For all IEC 60559 extended (and
interchange) formats, emin = 3 emax .

Simply put, `x` is the next precision up, i.e: `alias _Float32x = double;`
